src/info_html.R

```diff
@@ -50,8 +50,10 @@ renderMethodsHtml <- function() {
         
        tags$ul(tags$li("The methodoloy from ", tags$a("Russell et al (2020)", href = urls$russel_etal_2020),
                        " uses the county-level case fatality rate (CFR) to estimate the percentage of cases that are not reported. ",
-                       "Because the CFR is noisy for counties with few cases, we use a ", 
-                       tags$a("Bayesian adjustment", href = urls$conjugate_priors), " to shrink the county CFR ",
+                       "Because under-reporting has decreased since the beginning of the pandemic (in part due to increased access to testing), ",
+                       " we use a 90-day sliding window for cases and deaths to calculate the county-level CFR. ",
```

Comment:
Is there a difference between a "moving" and "sliding" window? 

For the 14 day case count, we use the word "moving" to describe the action. I am wondering if the process of moving and sliding are the same, then maybe we should use the same wording for both.

Reply:
That makes sense! I've replaced "sliding" with "moving".